### PR TITLE
Add equipment sponsor section

### PR DESCRIPTION
**Pull request by mzampetakis**
Please feel free to contribute on this or discuss it before merging. 
We might need some more equipment or it might be not so necessary.

### CODE REVIEW

File: Sponsors.md [hunk 1]
```diff
@@ -80,6 +79,18 @@ The list below describes the awareness benefits that our Logo sponsor receives i
 * “Thank you” mention in facebook & Twitter page post,
 * “Thank-you” message by organizers during Meetup (not by company/individual).
 
+## Equipment Sponsor
+As we are not close to the local community of Crete we try to live stream our monthly meetups and supply VoD through Youtube. FOr this reason we use equipment which mosto of the times is not suitable for our need.
```
Comment (DaKnOb) [inline Sponsors.md]: YouTube*
For*
most of*
needs*

File: Sponsors.md [hunk 2]
```diff
@@ -80,6 +79,18 @@ The list below describes the awareness benefits that our Logo sponsor receives i
...
+
+So we want some equipment to support even better our live stream capabilities such as two 1080p usb cameras and a deskptop micorphone.
```
Comment (DaKnOb) [inline Sponsors.md]: So we would like to upgrade our equipment, to improve our live streaming capabilities. The initial equipment required is:

* Logitech C922 Pro [80 EUR]
* Logitech C922 Pro [80 EUR]
* Desktop Microphone [50 EUR]

File: Sponsors.md [hunk 3]
```diff
@@ -80,6 +79,18 @@ The list below describes the awareness benefits that our Logo sponsor receives i
...
+
+The list below describes the awareness benefits that our equipment sponsor receives in return for every 100 EUR offer:
```
Comment (DaKnOb) [inline Sponsors.md]: equipment sponsor receives in return for sponsoring two or all of the items above:

File: Sponsors.md [hunk 4]
```diff
@@ -80,6 +79,18 @@ The list below describes the awareness benefits that our Logo sponsor receives i
...
+* leaflets (can be left on table at entrance),
+* online banner on DevStaff website (with back-link do-follow),
+* “Thank-you” message by organizers during Meetup (not by company),
+* Sponsor’s slide in DevStaff Welcome presentation,
+* “Thank you” mention in facebook page post.
+
```
Comment (DaKnOb) [inline Sponsors.md]: * "Thank you" mention on DevStaff's Twitter
* "Thank you" mention on DevStaff's LinkedIn
* Logo on DevStaff's Sponsors Slide in Live Stream

File: Sponsors.md (hunk 1 above)
Comment (sonaht) [inline Sponsors.md]: ** For this reason we use equipment which most of **

File: Sponsors.md (hunk 2 above)
Comment (sonaht) [reply Sponsors.md]: I would add a stand as well or even two, for both cameras. A tripod for the first ...to be floor bound and a smaller one for the desktop.

Comment (DaKnOb) [reply Sponsors.md]: This is a good idea, but will make the entirety of the equipment more bulky.

Comment (DaKnOb) [reply Sponsors.md]: And if we have a camera in the desk, we will definitely need to make sure that we can have a USB cable that works all the way to the laptop that does the live stream.

**Comment by DaKnOb** [comment]
Can you make the required changes @mzampetakis ?

File: Sponsors.md (hunk 1 above)
Comment (mzampetakis) [reply Sponsors.md]: Fixed by #2b308bbb5a661f83a7deec7f2a009fb3a90f38fa.

File: Sponsors.md (hunk 3 above)
Comment (mzampetakis) [reply Sponsors.md]: Fixed by https://github.com/devstaff-crete/DevStaff-Heraklion/pull/413/commits/2b308bbb5a661f83a7deec7f2a009fb3a90f38fa

File: Sponsors.md (hunk 4 above)
Comment (mzampetakis) [reply Sponsors.md]: Fixed by https://github.com/devstaff-crete/DevStaff-Heraklion/pull/413/commits/2b308bbb5a661f83a7deec7f2a009fb3a90f38fa

We don't maintain LinkedIn but we maintain a Twitter account!

File: Sponsors.md (hunk 1 above)
Comment (mzampetakis) [reply Sponsors.md]: Fixed by https://github.com/devstaff-crete/DevStaff-Heraklion/pull/413/commits/2b308bbb5a661f83a7deec7f2a009fb3a90f38fa

File: Sponsors.md (hunk 2 above)
Comment (mzampetakis) [reply Sponsors.md]: Fixed by https://github.com/devstaff-crete/DevStaff-Heraklion/pull/413/commits/2b308bbb5a661f83a7deec7f2a009fb3a90f38fa

**Comment by mzampetakis** [comment]
All of the issues have been resolved but I would suggest discussing it more thoroughly however...

**Comment by mzampetakis** [comment]
Closed this as it is not relevant anymore.

**Pull request closed without merging**